src/lib/utils/helpers.ts

```diff
@@ -154,3 +154,10 @@ export const numToWord = (num) => {
       (num % 10 ? '-' + belowTwenty[(num % 10) - 1] : '')
     )
 }
+
+/**
+ * Convert newline characters to HTML <br> tags.
+ */
+export const newlinesToBr = (content: string): string => {
+  return content.replace(/\n/g, '<br>')
```

Comment:
Idle curiosity: Should this not be a self-closing tag? I remember `<br>` being weird. 🤔

Reply:
You might then ask, as my brain did, whether since we're feeding this HTML to React React will be unhappy. It doesn't seem to be though.